tests/handlers/test_profile.py

```diff
@@ -344,6 +347,11 @@ def test_avatar_fetching_metadata_right_source(self) -> None:
                 user_id=UserID.from_string("@whatever:test"),
             )
         )
+        res = self.get_success(
+            self.handler.check_avatar_size_and_mime_type("mxc://test/local")
+        )
+        self.assertTrue(res)
```

Comment:
again: this check seems to pass with no code changes. In order to check that we have fixed the original bug, you need to use a `server_name` with an explicit port.